
# Part 1 - Brainstorm: Statistics

## Distributions
## Statistical Models
## Methods for Estimation
## Methods for Hypothesis Testing

# Part 2 - Brainstorm: Technologies in Biology

## Microarray
## ...

## In-class exercise 1 (15 minutes)

### Technologies: 

```{r}
techs <- c("microarray", "rna-seq", "dna-seq", 
           "bisulphite-seq", "cytometry", "mass-spec", 
           "10x-chromium", "oxford-nanopore")

s <- sample(length(techs))
data.frame(row=s, techs)
```

### Task: 
#### produce a 2-3 point summary of "how it works"
#### links to a few (<5) good resources
#### create a markdown file for it and upload to README.me in "group assignment" repo

# Part 3 - Brainstorm: Applications in genomics 

# Part 4 - Brainstorm: Linking Technologies to Applications to Statistics

## e.g., microarray -> gene expression -> normally distributed (log intensities)

# Part 5 - Brainstorm: Methods/algorithms in genomics associated to Computer Science

# Part 6 - Pick a "technology" (from above, from [1] or otherwise) to briefly describe

## Exercise 2 (in groups of 1-3): 
### Goal: 
#### write ~2 sentences about what the method does
#### again, make the link (technology -> application -> statistics)
#### list the github usernames of everyone in your group
#### submit a pull request to brainstorm\_modified.md



_Vivian Leung_ (vivianleung)


##### __IMS-MDA Seq__

Seth-Smith, H.M., Harris, S.R., Scott, P., Parmar, S., Marsh, P., Unemo, M., Clarke, I.N., Parkhill, J. and Thomson, N.R., 2013. Generating whole bacterial genome sequences of low-abundance species from complex samples with IMS-MDA. _Nature protocols_, 8(12), p.2404.

IMS-MDA-Seq, immunomagnetic separation (IMS) and multiple displacement amplification (MDA, is a method that enables affordable whole-genome sequencing of complex and low-abundance\* samples directly and without requiring bacterial culturing. Enrichment is achieved using antibodies conjugated to magnetic beads that bind specifically to target cells. Then, amplified genomic DNA is generated by MDA, quantified and then sequenced using high-throughput sequencing methods.

__technology__&nbsp; \{rightlongarrow\} &nbsp; __application__&nbsp; $\{\rightlongarrow\} &nbsp; __statistics__

IMS-MDA-Seq &nbsp; \{rightlongarrow\} &nbsp;  single-cell sequencing, whole-genome and heterogeneity analysis &nbsp; \{rightlongarrow\} &nbsp; principal component analysis, Burrows-Wheeler transform, binomial exact test 


[1] [https://liorpachter.wordpress.com/seq/](https://liorpachter.wordpress.com/seq/)



